firestore-bigquery-export/scripts/gen-schema-view/src/index.ts

```diff
@@ -123,40 +234,109 @@ interface CliConfig {
   bigQueryProjectId: string;
   datasetId: string;
   tableNamePrefix: string;
+  collectionPath?: string;
+  schemaDirectory?: string;
   schemas: { [schemaName: string]: FirestoreSchema };
+  useGemini?: boolean;
+  agentSampleSize?: number;
+  googleAiKey?: string;
 }
 
 async function run(): Promise<number> {
-  // Get all configuration options via inquirer prompt or commander flags.
   const config: CliConfig = await parseConfig();
 
-  // Set project ID so it can be used in BigQuery intialization
   process.env.PROJECT_ID = config.projectId;
-  // BigQuery actually requires this variable to set the project correctly.
   process.env.GOOGLE_CLOUD_PROJECT = config.bigQueryProjectId;
 
-  // Initialize Firebase
   if (!firebase.apps.length) {
     firebase.initializeApp({
       credential: firebase.credential.applicationDefault(),
       databaseURL: `https://${config.projectId}.firebaseio.com`,
     });
   }
 
-  // @ts-ignore string not assignable to enum
-  if (Object.keys(config.schemas).length === 0) {
-    console.log(`No schema files found!`);
-  }
   const viewFactory = new FirestoreBigQuerySchemaViewFactory(
     config.bigQueryProjectId
   );
-  for (const schemaName in config.schemas) {
-    await viewFactory.initializeSchemaViewResources(
-      config.datasetId,
-      config.tableNamePrefix,
-      schemaName,
-      config.schemas[schemaName]
-    );
+
+  if (config.useGemini) {
```

Comment:
should we check if the schema file already exist and exit?

Reply:
yes, good catch